### PR TITLE
Add grpc-1.0 and grpcClient-1.0 feature descriptions

### PR DESCRIPTION
**Pull request by wtlucy**
For #10088 and #10089

### CODE REVIEW

File: dev/com.ibm.websphere.appserver.features/visibility/public/grpcClient-1.0/resources/l10n/io.openliberty.grpcClient-1.0.properties [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature provides a gRPC client which can be used by applications to consume gRPC services.
```
Comment (ManasiGandhi) [inline dev/com.ibm.websphere.appserver.features/visibility/public/grpcClient-1.0/resources/l10n/io.openliberty.grpcClient-1.0.properties:17]: Change,
"This feature provides a gRPC client which can be used by applications to consume gRPC services."
to,
"This feature provides a gRPC client, which can be used by applications to use gRPC services."

Comment (wtlucy) [reply dev/com.ibm.websphere.appserver.features/visibility/public/grpcClient-1.0/resources/l10n/io.openliberty.grpcClient-1.0.properties:17]: Hi @ManasiGandhi - isn't "which can be **used** by applications to **use** gRPC services" a little redundant?  Is there a better translation-friendly verb we can use?

Comment (ManasiGandhi) [reply dev/com.ibm.websphere.appserver.features/visibility/public/grpcClient-1.0/resources/l10n/io.openliberty.grpcClient-1.0.properties:17]: "This feature provides a gRPC client, which can be used by applications to call gRPC services."

**Comment by wtlucy** [comment]
#build

**Comment by wtlucy** [comment]
#run-libby-bot

**Pull request merged**